content/blog/2019/07/2019-07-21-Jenkins-code-coverage-diff-for-pull-request.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Jenkins code coverage diff in pull requests"
+tags:
+- plugins
+- codecoverage
+author: asavanchuk
```

Comment:
Social media mentions of a blog post will generate more traffic if the page includes an opengraph image that visually highlights the topic of the post.  Since the post already includes this image of a code coverage increase, we can reuse the image for social media and increase the chances that others will visit the blog post.
```suggestion
author: asavanchuk
description: Report code coverage changes in pull requests
opengraph:
  image: /images/post-images/2019-07-21-Jenkins-code-coverage-diff-for-pull-request/increase.png
```

Reply:
Thank you! Fixed the issues.